docs/content/en/release-notes/_index.md

```diff
@@ -110,4 +144,4 @@ To submit documentation comments or edits, see [Contribution guidelines]({{< rel
 
 Read the K8ssandra [FAQs]({{< relref "faqs" >}}) - for starters, how to pronounce "K8ssandra." 
 
-If you're impatient, jump right in with the K8ssandra [install]({{< relref "install" >}}) steps.
+If you're impatient, jump right in with the K8ssandra [install]({{< relref "install" >}}) steps for [local]({{< relref "install/local" >}}), Google Kubernetes Engine ([GKE]({{< relref "install/gke" >}})), and Amazon Elastic Kubernetes Service ([EKS]({{< relref "install/eks" >}})) platforms; plus a signup form for an upcoming install topic detailing K8ssandra on Microsoft Azure Kubernetes Service ([AKS]({{< relref "install/aks" >}})).
```

Comment:
I believe the AKS docs may be live by the time this PR gets merged. Let's track this change here.

Reply:
Updated reference to Install > AKS topic with more recent commit.